pennclubs/settings/base.py

```diff
@@ -107,6 +109,8 @@
     'django.contrib.auth.backends.ModelBackend',
 )
 
+AUTH_USER_MODEL = 'users.Person'
```

Comment:
Are we using a custom user model for clubs? Might be more of an @ArmaanT question, but my preference for Django is to stick to the base user class and create one to one with a custom user model, so that Django admin can work normally out of the box.

Reply:
I don't think we need a custom user model. See my comment [above](https://github.com/pennlabs/clubs-backend/pull/6#discussion_r314948702).

Reply:
Agreed, changed back to default user model.